test/plugins/test/autolink.js

```diff
@@ -109,13 +109,15 @@ describe('AutoLink', function() {
 
 	it('should not create a link if last word is not a valid URL', function() {
 		testLink.call(this, {
-			expected: '<p>invalid link ww.liferay.com</p>',
+			expected:
+				'<p>invalid link <a href="http://ww.liferay.com">ww.liferay.com</a></p>',
```

Comment:
So none of these tests make sense any more, do they? They say "should not create a link" but evidently we now do because `ww.liferay.com` *does* look like a URL.

Reply:
I modified the test descriptions, tell me what you think, it still might seem strange in some cases  :man_shrugging: